modules/apps/transfer/ibc_module.go

```diff
@@ -10,7 +10,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	capabilitytypes "github.com/cosmos/ibc-go/modules/capability/types"
-	"github.com/cosmos/ibc-go/v7/modules/apps/transfer/keeper"
+	transferkeeper "github.com/cosmos/ibc-go/v7/modules/apps/transfer/keeper"
```

Comment:
which linter is this? I actually like this change, I'm aware the convention is to use `keeper` everywhere, cc @damiannolan  any thoughts?

Reply:
I guess the more widely used convention with sdk modules is that local keeper is `keeper` and external keepers are `xyzkeeper`

Reply:
all of this is revive, but I suspect that this had to do with import shadowing.

Reply:
If we do it here, would be nice to do it for `icahost` too. Currently there's a mismatch between em.

Reply:
I think I know the solution, thanks guys for the thorough review of this, since the goal is to set down some lasting standards :)